### PR TITLE
Fix incorrect imports

### PR DESCRIPTION
**Pull request by egaluza**

### CODE REVIEW

File: js/core/renderer.d.ts [hunk 1]
```diff
@@ -0,0 +1,256 @@
+interface JQueryCoordinates {
```
Comment (IlyaKhD) [inline js/core/renderer.d.ts]: I suggest renaming all those `jQueryXXX` types to `jQueryLikeXXX` e.g.:
```suggestion
interface JQueryLikeCoordinates {
```

Maybe we should move all of them to a separate folder.

File: js/core/renderer.d.ts [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+    
+    attr(attributeName: string, value: string|number|null): JQuery;
+    
+    attr(attributeName: string, func: (index: number, attr: string) => string|number): JQuery;
+    
+    hasClass(className: string): boolean;
+    
+    html(htmlString: string): JQuery;
+    
+    html(func: (index: number, oldhtml: string) => string): JQuery;
```
Comment (IlyaKhD) [inline js/core/renderer.d.ts]: It seems that this overload is not implemented:

```js
initRender.prototype.html = function(value) {
    if(!arguments.length) {
        return this[0].innerHTML;
    }

    this.empty();

    if(typeof value === 'string' && !isTablePart(value) || typeof value === 'number') {
        this[0].innerHTML = value;

        return this;
    }

    return this.append(parseHTML(value));
};
```

Comment (IlyaKhD) [reply js/core/renderer.d.ts]: I think we should check for unsupported declarations and remove them.
And see if it breaks compilation 🍿

File: js/core/renderer.d.ts (hunk 1 above)
Comment (nightskylark) [reply js/core/renderer.d.ts]: I'd prefer to avoid `jquery` word usage everywhere except the `integrations/jquery` folder.

Comment (egaluza) [reply js/core/renderer.d.ts]: Leave part of the name XXX

File: js/core/renderer.d.ts (hunk 2 above)
Comment (egaluza) [reply js/core/renderer.d.ts]: Done

**Review by dxvladislavvolkov — DISMISSED** [review]

File: js/core/renderer.d.ts [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    
+    wrapInner(wrappingElement: JQuery|Element|string): JQuery;
+    
+    each(func: (index: number, elem: Element) => boolean | void): JQuery;
+    
+    get(index: number): Element;
+
+    index(selector: string|JQuery|Element): number;
+    
+    add(...elements: Element[]): JQuery;
```
Comment (IlyaKhD) [inline js/core/renderer.d.ts]: It seems that our implementation differs from jQuery:
```js
initRender.prototype.add = function(selector) {
    const targets = renderer(selector);
    const result = this.toArray();

    for(let i = 0; i < targets.length; i++) {
        const target = targets[i];
        if(result.indexOf(target) === -1) {
            result.push(target);
        }
    }

    return renderer(result);
};
```

**Review by nightskylark — APPROVED** [review]

**Pull request merged**